### PR TITLE
[refactor] Support for openwisp-utils versions 0.6 and 0.7 #77

### LINKED ISSUE #77: [release] Release django-loci 0.4.0
Let's [release openwisp-utils 0.7.0](https://github.com/openwisp/openwisp-utils/issues/157) first and update the requirement files to support both 0.6 and 0.7 (to avoid conflict with other modules having older versions).

Then we can release django-loci 0.4.0.

### PR DESCRIPTION
**Pull request by devkapilbansal**
Updated requirements to support both openwisp-utils versions, i.e. 0.6 and 0.7

Closes #77

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ attrs>=17.4.0
 pytest-django>=3.4.2,<3.10.0
 pytest-asyncio>=0.10.0,<0.15.0
 pytest-cov>=2.7.0,<2.11.0
-openwisp-utils[qa]~=0.6.0
+openwisp-utils[qa]>=0.6.0,<=0.7.0
```
Comment (pandafy) [inline requirements-test.txt:6]:
```suggestion
openwisp-utils[qa]>=0.6.0,<0.8
```
This way will support all minor versions of openwisp-utils 0.7

File: requirements.txt [hunk 2]
```diff
@@ -2,5 +2,5 @@ django>=2.2,<3.2
 django-leaflet>=0.23,<0.28
 channels>=2.3.0,<2.5.0
 Pillow>=6.2.0,<=8.1.0
-openwisp-utils~=0.6.0
+openwisp-utils>=0.6.0,<=0.7.0
```
Comment (pandafy) [inline requirements.txt:5]:
```suggestion
openwisp-utils[qa]>=0.6.0,<0.8
```
This way will support all minor versions of openwisp-utils 0.7

**Review by pandafy — CHANGES_REQUESTED** [review]
@devkapilbansal can break these changes into different commits? One for refactoring and other for upgrading the dependency?

**Review by pandafy — APPROVED** [review]
LGTM :+1: 

Merging deferred to @nemesisdesign

**Review by nemesisdesign — APPROVED** [review]
Thanks @devkapilbansal @pandafy :+1:

**Pull request merged**